content/general/concepts/big-o-notation/big-o-notation.md

```diff
@@ -46,4 +46,4 @@ def foo(list1, list2):
 foo(["Hello", "World"], ["Code", "Ninjas"])
 ```
 
-For each item of `list1` in the outer loop, each item in `list2` is traversed in an inner loop. The Big-O notation for the `foo()` function would be $O(n^{2})$.
+For each item of `list1` in the outer loop, each item in `list2` is traversed in an inner loop. The Big-O notation for the `foo()` function would be _O(n^2)_.
```

Comment:
So can we say that the first loop took *O(n)* time and the outer loop executed this loop *n* times, so the total time complexity is *n* \* *O(n)* = *O(n^2)*? I feel like this would be more helpful.